jaeger-thrift/build.gradle

```diff
@@ -40,7 +43,8 @@ checkstyleTest.enabled = false
 shadowJar {
     baseName = 'jaeger-thrift'
     relocate 'org.apache.thrift', 'org.shadow.apache.thrift92'
-    classifier 'thrift92'
+    relocate 'okhttp', 'jaeger.okhttp'
```

Comment:
nit: we should maybe have a consistent relocate pattern - so possibly prefix jaeger as here, so update the thrift one to `jaeger.org.apache.thrift` (without the 92)?
btw - might also be worth looking at the other dependencies included in this shaded jar, as they may be should be relocated aswell to avoid any potential clashes  - or possibly this should be a separate PR.

Reply:
> or possibly this should be a separate PR

That would be my preference.

Reply:
https://github.com/jaegertracing/jaeger-client-java/issues/450